website/src/docs/hotchocolate/v14/migrating/migrate-from-13-to-14.md

```diff
@@ -124,6 +124,20 @@ builder.Services
     .AddGlobalObjectIdentification();
 ```
 
+## IIdSerializer replaced by INodeIdSerializer
+
+Previously, you could grab the `IIdSerializer` from your dependency injection container to manually parse and serialize globally unique identifiers (GID).
+As part of the changes to the GID format mentioned above, the `IIdSerializer` interface has been renamed to `INodeIdSerializer`.
+
+The methods used for parsing and serialization have also been renamed:
+
+| Before                             | After                                                                                    |
+| ---------------------------------- | ---------------------------------------------------------------------------------------- |
+| `.Deserialize("<gid-value>")`      | `.Parse("<gid-value>", typeof(string))` where `string` is the underlying type of the GID |
+| `.Serialize("MyType", "<raw-id>")` | `.Format("MyType", "<raw-id>")`                                                          |
```

Comment:
The 3 parameter version of serialize should be included here as well.

```
| `.Serialize("SchemaName", "MyType", "<raw-id>"`)` | `???` |
```

It looks like the named schema feature was dropped?